package.json

```diff
@@ -12,7 +12,7 @@
     "a11y"
   ],
   "engines": {
-    "node": ">=6"
+    "node": ">=8.7.0"
```

Comment:
https://github.com/stylelint/stylelint/blob/b35c55f6144d0cb03924498f275cc39c39de3816/package.json#L38-L40

Reply:
I agree, we must comply with the main project (Stylelint) in there.